### ISSUE TITLE
How to upgrade my meanjs environment to the latest version?

### ISSUE THREAD

**Issue by kitce** [meanjs/mean#1050]
I am not sure which version I am using, may be 0.3.x.

I realized that now meanjs provide a grunt task `prod` for running the application in production environment, I hope to enjoy this feature rather than just running `node server.js`.

Actually, my application is not ready for production yet, but test environment. But without this feature, I have to change `NODE_ENV` to `test` or `production` before I execute `node server.js` to start the application, it's quite inconvenient.

In short, is it safe for upgrading the meanjs environment? And how?

**Comment by lirantal** [comment]
Depends what you call by "safe" :)
A lot has changed in the MEAN.JS structure of files/folders and also how the application is structured itself from 0.3 to 0.4.

There's no 'path of upgrade' if that's what you're looking for.
What I suggest doing is cloning the latest `master` branch of MEAN.JS and seeing how you can copy&paste old files functionality from your previous MEAN.JS app to the new one you cloned.

**Comment by kitce** [comment]
Copy&paste would be "safe" to upgrade the meanjs environment , I will try it when 0.4.x is stable, thank you for the advice. I am closing this.

Looking forward to see the better ways to do this in future.
